File: backend/webapicsharp/Modelos/Trabajador.cs
namespace webapicsharp.Modelos
{
    public class Trabajador : Empleado
    {
        public string? Horario { get; set; }

        public Trabajador(int id, string nombre, string cedula, string correo, string direccion, string telefono, string contrasena, string codigoDeEmpleado, string horario)
                : base(id, nombre, cedula, correo, direccion, telefono, contrasena, codigoDeEmpleado)
        {
            Horario = horario;
        }
    }

    public class TrabajadorDto : EmpleadoDto
    {
        public string? Horario { get; set; }
    }
}

File: frontend/src/interfaces/trabajador.ts
export interface Trabajador {
    Id: number
    Nombre: string
    Cedula: string
    Correo: string
    Direccion: string
    Telefono: string
    CodigoDeEmpleado: string
    Horario: string
}

export interface TrabajadorFormData {
    nombre: string
    cedula: string
    correo: string
    direccion: string
    telefono: string
    contrasena: string
    codigoDeEmpleado: string
    horario: string
}
